OrbitCore/UserDefinedCaptureDataTest.cpp

```diff
@@ -0,0 +1,101 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <gtest/gtest.h>
+
+#include <string>
+
+#include "UserDefinedCaptureData.h"
+#include "capture_data.pb.h"
+
+using orbit_client_protos::FunctionInfo;
+
+FunctionInfo CreateFunctionInfo(const std::string& function_name, uint64_t function_address) {
+  FunctionInfo info;
+  info.set_name(function_name);
+  info.set_pretty_name(function_name);
+  info.set_loaded_module_path("/path/to/module");
+  info.set_address(function_address);
+  info.set_size(16);
+  info.set_file("file.cpp");
+  info.set_line(13);
+  return info;
+}
+
+TEST(UserDefinedCaptureData, InsertFrameTrack) {
+  UserDefinedCaptureData data;
+  FunctionInfo info = CreateFunctionInfo("fun0_name", 0);
+  data.InsertFrameTrack(info);
+  EXPECT_TRUE(data.ContainsFrameTrack(info));
+}
+
+TEST(UserDefinedCaptureData, InsertFrameTrackDuplicateFunctions) {
```

Comment:
Depending on the decision on the above comment, this testcase might be irrelevant.

Reply:
Done. Actually added an EXPECT_DEATH check here.